extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -260,6 +259,15 @@ class AmpBridPlayer extends AMP.BaseElement {
     if (params[2] == 'trigger') {
       if (params[3] == 'ready') {
         this.playerReadyResolver_(this.iframe_);
+      } else if (params[3] == 'play') {
+    	if (this.element.hasAttribute('data-outstream')) {
+          setStyles(this.element, {transition: 'height 2s',
```

Comment:
`data-outstream` will only play an `Ad`(if ad tag URL returns it). We have multiple publisher's requests in which they want unit collapsed in case when there is no `Ad` or when `Ad` is finished. As I mentioned before, we saw that `teads` unit has collapse behavior, that was the page our publishers used as example in their requests. 

In highlighted code, we just added`CSS` transition when `Ad` is played to make collapse transition more smoothly.